### ISSUE TITLE
Doesn't work with python-pyalsa 1.1

### ISSUE THREAD

**Issue by daev3000** [MaurizioB/Bigglesworth#5]
Hi. I'm running debian sid, and Bigglesworth only works after I downgrade python-pyalsa down to version 1.0.29 (from the Debian Jessie repository). I'd love to be able to run it using python-pyalsa 1.1.6.

**Comment by MaurizioB** [comment]
Hi. I've not upgraded pyalsa yet, as I'm trying to fix everything else with a stable release for the current implementation (I had to create a compatibility layer that works for both rtmidi and pyalsa).
As soon as this stage is completed, I'll certainly try and check out the new version.
If you don't mind, in the meantime, can you be more specific about what doesn't work? If you can share the console output too, it would be very helpful.
Also, are you using the old "stable" release or the one in the dev branch?

**Comment by daev3000** [comment]
I'm using the old "stable" release, which has been has been fairly stable in my time using it. I'm not sure if this issue affects the newer code; if you want to close the issue and keep focusing on the newer version, that's fair.

Anyway, with the old release and the newer python-pyalsa, the console output is as follows:

Traceback (most recent call last):
  File "./BigglesworthApp", line 4, in <module>
    import bigglesworth
  File "/home/daev/builds/Bigglesworth/bigglesworth/__init__.py", line 12, in <module>
    from midiutils import *
  File "/home/daev/builds/Bigglesworth/bigglesworth/midiutils.py", line 237, in <module>
    _alsa_event = getattr(alsaseq, 'SEQ_EVENT_{}'.format(alsa_str))
AttributeError: 'module' object has no attribute 'SEQ_EVENT_USR0'

**Comment by MaurizioB** [comment]
Don't worry, the issue will remain open until I can confirm that it doesn't apply to the new branch.
In the meantime, you can try and comment line 223 in the "midiutils.py" file, which is this one: 

                    1073741823: ('USR0', 'ANY'),

put a leading "#", just like the line before.
Save the file and run it again, if we're lucky everything will be fine...

**Comment by daev3000** [comment]
Commenting out line 223 works for me. Thanks!

**Comment by MaurizioB** [comment]
We got lucky! ;-)
Good to know, anyway, thank you. I'll comment that line on the new code also, I actually added all that stuff for (possible) future compatibility, but I suppose we can already say that that kind of MIDI event is not useful at all, since they removed themselves.
I'll mark this as closed and solved, as I'll move the dev branch to stable as soon as possible (less than a week), which will automatically solve this issue.
Thank you again!
